### DISCUSSION TITLE
Modification of UCRM/Twillio SMS Plugin.

### DISCUSSION THREAD

**Discussion by Githuimiatu** [community/community#38310] [Apps, API and Webhooks]
Hi all.
I would like to modify  the current UCRM/Twilio SMS plugin. Am convinced that if I was to change a few parameters like the API credentials and URL to that of my service provider, the plugin would work just fine. I have however been unable to locate the URL where CRM posts the Message. Kindly assist. Your support is much appreciated.

**Comment by ossanna16** [comment]
Hi @Githuimiatu,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Twilio. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
